### DISCUSSION TITLE
Earning honor and completed katas not matched with real completed katas

### DISCUSSION THREAD

**Discussion by YanaSheri** [codewars/codewars.com#3298] [General]
Hi there,

I'm new to Codewars and absolutely confused. 
I need to earn 100+ honor for a few days (for  new role) but I don't understand what's going on.
I'm training Python, resolved 10+ tasks but still have honor 5 and completed katas 2. 
What does it mean? Why I didn't receive more honor and why completed katas only 2?

Please don't lough :) I just need to understand if I spend 24hours here can I earn minimum 50 honor?

Please give me advice how to earn more honor.

And 1 additional question: what does it mean - green/yellow/blue Python logo? Maybe it's a key?

Thanks

**Comment by hobovsky** [comment]
See if Codewars documentation helps, especially [Getting started](https://docs.codewars.com/getting-started/solving-kata) and [How to train](https://docs.codewars.com/training/training-example) articles. You can also get help on Codewars Discord, in `#beginners` or `#help-solve` channels.

It is also difficult to check what is going on without knowing your Codewars user name.

**Reply by hobovsky** [reply]
About colors:
- blue means that Python is a language you are interested with, but you did not solve or try the kata yet.
- yellow means that you tried to solve, but did not solve yet.
- green means that you solved the kata.

**Reply by YanaSheri** [reply]
I'm Yanchik3in1
Thank you!
I fixed my issue - I found that I should click "Submit" after all :)))
